app/src/test/java/com/techcourse/dao/UserDaoTest.java

```diff
@@ -22,6 +23,12 @@ void setup() {
         userDao.insert(user);
     }
 
+    @AfterEach
+    void tearDown() {
+        final JdbcTemplate jdbcTemplate = new JdbcTemplate(DataSourceConfig.getInstance());
+        jdbcTemplate.execute("TRUNCATE TABLE `users` RESTART IDENTITY");
```

Comment:
`jdbcTemplate.execute("TRUNCATE TABLE `users` RESTART IDENTITY");`부분은 테스트를 위한 코드 같은데,
```suggestion
        jdbcTemplate.update("TRUNCATE TABLE `users` RESTART IDENTITY");
```
와 같이 써보는 것은 어떨까요>?

Reply:
이레 말씀대로 테스트를 위한 코드는 맞습니다! 
하지만 굳이굳이 `execute`로 분리한 이유는 `update`는 DML (insert, update ,delete) 작업을 수행할 때 사용할 수 있는 메서드 입니다. 하지만 찾아보니 `truncate`는 DDL로 취급이 된다고 하더라구요!

그래서 이러한 DDL을 실행시키는 메서드가 따로 있으면 더 좋을 것 같다고 생각했고, 이를 위해 "SQL문 종류에 상관없이 실행시킬 수 있는 메서드" 의 의미로 `execute` 메서드를 추가했습니다.

이번 미션에서는 테스트에서 밖에 쓰이지 않더라도 DB에 쿼리를 날리는 역할을 담당하는 JdbcTemplate의 특성 상, 충분히 있어도 되는 메서드라고도 생각했구요..!

다만 확실히 이레 말씀처럼 현재로서는 테스트를 위한 메서드라 조금 신경이 쓰이긴 하네요 ㅠㅠ 이레는 어떻게 생각하시나요?

Reply:
사실 테스트를 위한 코드는 좋아보이지는 않습니다.
그리고 실상 내부적으로는 executeUpdate()라는 메서드를 사용함으로써, update()메서드와 완전히 동일한 코드가 진행되는 것 같아서요. 
굳이 execute()라는 메서드를 사용할 필요가 없는 것 같다고 여겨집니다!

Reply:
public int update(final String sql, final Object... args) {
        return executeInternal(sql, pstmt -> executeUpdate(pstmt), args);
    }

Reply:
맞습니다 사실 완전히 동일한 동작을 하고 있으니까요 ㅎㅎ
이 부분은 `execute`를 없애고 `update`로 변경했습니다!